techniques/system/common/1.0/update.st

```diff
@@ -312,12 +323,9 @@ bundle agent update_reports
     config::
       "@@Common@@log_repaired@@&TRACKINGKEY&@@Update@@None@@${g.execRun}##${g.uuid}@#Node's policy (CFEngine promises) updated";
 
-    rudder_promises_generated_repaired|config|rudder_tools_updated|rudder_ncf_common_updated|rudder_ncf_local_updated|server_ok|executor_ok::
+    config|rudder_tools_updated|rudder_ncf_common_updated|rudder_ncf_local_updated|server_ok|executor_ok::
```

Comment:
I removed ```rudder_promises_generated_repaired``` from the condition to print a ```result_repaired``` here because it just means that the ```rudder_promises_generated``` file was updated. This, by itself, doesn't functionnaly consitute an update. What does, is if the promises are then updated, but the ```config``` class already covers this condition.